.travis.yml

```diff
@@ -1,35 +0,0 @@
-sudo: false
-language: elixir
-git:
-  depth: 3
-elixir:
-  - 1.8
-  - 1.9
-  - 1.10
-otp_release:
-  - 21.3
-  - 22.3
-  - 23.0
-matrix:
-  exclude:
-    - elixir: 1.8
-      otp_release: 23.0
-    - elixir: 1.9
-      otp_release: 23.0
-env:
-  - MIX_ENV=test VERBOSE_TESTS=true
-install:
-  - mix local.rebar --force
-  - mix local.hex --force
-  - mix deps.get
-  - mix eqc.install --mini --force
-before_script:
-  - wget https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protoc-3.12.3-linux-x86_64.zip
-  - unzip protoc-3.12.3-linux-x86_64.zip
-  - export PATH=$PATH:$PWD/bin/
-script:
-  - mix test
-  - mix test.integration
-  - mix hex.build
-  # Only run for latest version
-  - '[[ "$TRAVIS_ELIXIR_VERSION" =~ 1.[8-9] ]] || mix format --check-formatted'
```

Comment:
I noticed we don't check formatting in the new pipeline anymore. Should we bring it back?

Reply:
Let's do it yes, but in a later PR :)

Reply:
@britto actually can we also add `mix compile --warnings-as-errors` and `mix deps.unlock --check-unused` to the `lint: true` CI step as part of linting?

Reply:
Cool, I'll give it a shot.